apoiase/app.py

```diff
@@ -20,17 +20,19 @@ def home():
     if response.status_code == 200:
         page = json.loads(response.text)
         users = page.get('supports', {}).get('users', [])
-        names = map(serializer, users)
-        names = [name for name in names if name]
+        users = filter(filter_private_supporters, users)
+        names = list(map(serializer, users))
         return jsonify(names)
 
 
-def serializer(users):
-    private=users.get('privateSupport')
-    if not private:
-        return dict(name=users.get('_id').get('name'))
+def filter_private_supporters(apoiase_users):
```

Comment:
If `privateSupport` is alteady a boolean it's even easier, yay!

But IMHO it's not this function role to filter users. The bit that actually filter users is the `filter(is_private_user, users)` above, isn't it? 

Given that user already has a boolean for that, we can even go for `filter(lambda u: u.get('privateSupport'), users)` BTW…